src/platform/Linux/DnssdImpl.cpp

```diff
@@ -988,7 +1047,10 @@ CHIP_ERROR ChipDnssdResolve(DnssdService * browseResult, chip::Inet::InterfaceId
                                             browseResult->mAddressType, Inet::IPAddressType::kAny, interface, callback, context);
 }
 
-void ChipDnssdResolveNoLongerNeeded(const char * instanceName) {}
+void ChipDnssdResolveNoLongerNeeded(const char * instanceName)
+{
+    MdnsAvahi::GetInstance().StopResolve(instanceName);
```

Comment:
StopResolve can handle nullptr?

Reply:
I see no evidence that ChipDnssdResolveNoLongerNeeded is supposed to allow nullptr.